app/helpers/anchor/tailwind_constants.rb

```diff
@@ -0,0 +1,80 @@
+module Anchor
+  module TailwindConstants
+    ERROR = %w(
+      bg-critical-subdued
+      rounded
+      py-1
+      px-2
+      mt-1
+      text-sm
+      text-critical
+    ).freeze
+
+    HINT = %w(
+      mt-2
+      text-sm
+      text-gray-500
```

Comment:
This color will go away in https://github.com/BuoySoftware/anchor_view_components/pull/69 - let’s swap for…

```suggestion
      text-secondary
```

Reply:
Done.